src/repository/user.local.pg.repository.ts

```diff
@@ -0,0 +1,151 @@
+import Database from './database.interface';
+import UserRemote from '../model/entity/user.remote.type';
+import UserLocal from '../model/entity/user.local.type';
+import ListUser from '../model/request/list.user.type';
+
+const { PreparedStatement } = require('pg-promise');
+
+export const getUserByName = (database: Database) => async (username: string): Promise<UserLocal> => database.handler.oneOrNone(new PreparedStatement(
+  {
+    name: 'find-user-by-name',
```

Comment:
`$1`, `$2`, `$3` can get confusing for a large number of parameters, and it leads to fragile code (easy to swap the order by accident). Is there a better way to handle this offered by the library?

Reply:
Certainly, we can use the ```as.format()``` and ```helpers.insert()``` to have named parameters and still use prepared statements. I actually used those functions on more complex queries.